.github/workflows/fmt-and-lint.yml

```diff
@@ -29,6 +29,9 @@ jobs:
       - name: Build cache
         uses: ./.github/actions/build-cache
 
+      - name: Backup Cargo.lock
+        run: cp Cargo.lock Cargo.lock.bak
```

Comment:
We could simply run `cargo update -w --locked` which will fail if `Cargo.lock` isn't up-to-date.

Reply:
Cool, does the job! 
https://github.com/KomodoPlatform/komodo-defi-framework/actions/runs/12526268691/job/34938647572
```
      Adding web-time v1.1.0
error: the lock file /home/runner/work/komodo-defi-framework/komodo-defi-framework/Cargo.lock needs to be updated but --locked was passed to prevent this
```

commit https://github.com/KomodoPlatform/komodo-defi-framework/pull/2309/commits/4d83f94eadb7ea042875797955e3b0c2f8d4ba98